container/Dockerfile

```diff
@@ -1,7 +1,13 @@
+FROM registry.fedoraproject.org/fedora:35 as builder
```

Comment:
Not sure how long-lasting this should be, but `:latest` seems nicer

Reply:
done, thanks :)